modules/eks/github-actions-runner/README.md

```diff
@@ -8,6 +8,22 @@ by Summerwind and deployed by Cloud Posse's [actions-runner-controller](https://
 
 ### Current limitations
 
+The runner image used by Runner Sets contains [no more packages than are necessary](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners-with-actions-runner-controller/about-actions-runner-controller#about-the-runner-container-image)
+to run the runner. This is in contrast to the Summerwind implementation, which
+contains some commonly needed packages like `build-essential`, `curl`, `wget`,
+`git`, and `jq`, and the GitHub hosted images which contain a roubst set of tools.
+(This is a limitation of the official Runner Sets implementation, not this
+component per se.) You will need to
```

Comment:
```suggestion
component per se). You will need to
```

Reply:
No, the period goes inside the parenthesis when the parenthetical is a full sentence.